NYUDining/FbLoginViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  FbLoginViewController.swift
+//  NYUDining
+//
+//  Created by Jesus Leal on 6/28/16.
+//  Copyright © 2016 Ross Freeman. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class FbLoginViewController: UIViewController, FBSDKLoginButtonDelegate
+{
+    
+    @IBOutlet var btnFacebook: FBSDKLoginButton!
+//    @IBOutlet var btnFacebook: FBSDKLoginButton!
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()
+        configureFacebook()
+ 
+    }
+    
+    func loginButton(loginButton: FBSDKLoginButton!, didCompleteWithResult result: FBSDKLoginManagerLoginResult!, error: NSError!)
+    {
+        FBSDKGraphRequest.init(graphPath: "me", parameters: ["fields":"first_name, last_name, picture.type(large)"]).startWithCompletionHandler { (connection, result, error) -> Void in
+            
+            let strFirstName: String = (result.objectForKey("first_name") as? String)!
```

Comment:
You should never force unwrap an optional, even if you're practically certain it will never be nil. It's just safe practice to prevent unwanted crashes. This could be written as `if let strFirstName = result.objectForKey("first_name") as String`